Lesson9/merve/styles.css

```diff
@@ -0,0 +1,172 @@
+* {
+    box-sizing: border-box;
+  }
+  body{
+  max-width: 100%;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  font-size: 1em;
+  font-family:  medium-content-sans-serif-font, -apple-system, BlinkMacSystemFont,    "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-   serif;
+  font-weight: 400;
+  letter-spacing: 0.01em;
+  line-height: 1.4em;
+  }
+  h1{
+  font-family: sohne, "Helvetica Neue", Helvetica, Arial, sans-serif;
+   font-weight: 800;
+   font-size: 42px;
+   line-height: 52px;
+   letter-spacing: -0.011em;
+   color: #242424;
+   margin-top: 1.19em;
+   margin-bottom: 32px;
+  }
+  .first-box{
+   max-width: 700px;
+  }
+  .first-img{
+  border-radius: 50px;
+  position: absolute;
+  }
+  .second-img{
+  position: absolute;
+  margin-left: 30px;
+  margin-top: 15px;
+  border-radius: 50px;
+  border: 3px solid rgb(255, 255, 255);
+  }
+  a{
+  text-decoration: underline!important;
+  color:inherit;
+  font-weight: 500;
+  }
+  .profile, .next-info{
+  margin-left: 75px;
+  font-size: 14px;;
+  font-weight: 400;
+  line-height: 20px;
+  color: #242424;
+  }
+  .link {
+  color: inherit;
+  text-decoration: none;
+  }
+  #chart{
+  color:inherit;
+  }
+  .link:hover {
+  text-decoration: underline;
+  cursor: pointer;
+  }
+  .light {
+  color: #6B6B6B;
+  }
+  .btn {
+  color: rgb(66, 154, 53);
+  text-decoration: none;
+  cursor: pointer;
+  }
+  .divider {
+  border-bottom: 1px solid #6B6B6B;
+  padding-top: 30px;
+  }
+  .social-media-box{
+  height: 10px;
+  }
+  .clock{
+  display:flex;
+  align-items: center;
+  margin-top: 40px;
+  }
+  .images{
+  max-width:100%;
+  display: flex;
+  align-items:center;
+  justify-content: center;
+  }
+  figcaption{
+  display: flex;
+  align-items:center;
+  justify-content: center;
+  max-width: 728px;
+  margin-top: 10px;
+  margin-right: auto;
+  margin-left: auto;
+  text-align: center;
+  color: #6B6B6B;
+  font-size: 14px;
+  font-weight: 500;
+  }
+  .article, footer{
+  max-width: 60%;
+  margin: 50px auto;
+  letter-spacing: -0.003em;
+  line-height: 32px;
+  font-size: 20px;
+  font-family: source-serif-pro, Georgia, Cambria, "Times New Roman", Times, serif;
+  }
+  pre{
+  min-width: fit-content;
```

Comment:
Pre elements get too wide on small screens so add `overflow-x:scroll;` and remove min-width, margin-bottom and margin-top, they are not needed.
Its important how the content of the pre tags is formatted in HTML as everything inside it is shown on the screen.
<img width="1221" alt="Screenshot 2024-03-24 at 22 22 11" src="https://github.com/frontend-from-0/181223-exercises/assets/24791701/62b78552-5c2f-42aa-83ee-2215decb7ba9">

Reply:
"Its important how the content of the pre tags is formatted in HTML as everything inside it is shown on the screen." that you say.  Should i change the code in pre section ?